kayenta-web/src/main/java/com/netflix/kayenta/controllers/CanaryController.java

```diff
@@ -183,6 +183,7 @@ public CanaryExecutionStatusResponse getCanaryResults(
   List<CanaryExecutionStatusResponse> getCanaryResultsByApplication(
       @RequestParam(required = false) String application,
       @RequestParam(value = "limit", defaultValue = "20") int limit,
+      @RequestParam(value = "page", defaultValue = "1") int page,
```

Comment:
Just sanity checking -- should be 1-based rather than 0-based? :-)

Reply:
Yeah, it's [1-based](https://github.com/spinnaker/orca/blob/a3d2916ea1c5b3481ef85c4c0cd8ed0435effd96/orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/persistence/ExecutionRepository.java#L214)!